kodi_addon_checker/plugins/console_reporter.py

```diff
@@ -0,0 +1,22 @@
+from kodi_addon_checker.common import colorPrint
+
+from kodi_addon_checker.report import Reporter, Report, Record, reporter
+
+
+@reporter(config_property="console-reporter", enabled=True)
+class ConsoleReporter(Reporter):
+
+    def report(self, report: Report):
+        if type(report) is Record:
+            if report.information:
+                colorPrint(report, "34")
+            elif report.warning:
+                colorPrint(report, "35")
+            elif report.problem:
+                colorPrint(report, "31")
+            else:
+                print(report)
+        else:
+            print("\nChecking %s" % report.artifact_name)
```

Comment:
You can just use the info log level for this and drop this else

Reply:
Fixed by removing `print` and adding info logs before checking repos and add-ons. However, `else` is required for recursion.